addons/arsenal/functions/fnc_refresh.sqf

```diff
@@ -25,6 +28,21 @@ if (_updateItems) then {
     call FUNC(updateUniqueItemsList);
 };
 
+private _virtualItems = GVAR(currentBox) getVariable QGVAR(virtualItems);
+if (isNil "_virtualItems") exitWith {
+    [LLSTRING(noVirtualItems), false, 5, 1] call EFUNC(common,displayText);
```

Comment:
`STR_ACE_Arsenal_noVirtualItems` needs to be updated to `No virtual items available`